## DP: Tulip - fixes compile on MIPS64.
## DP: Path author: Peter Horton <pdh@colonel-panic.org>
## DP: Not sure whether this is needed or not, but we've had it for
## DP: quite a while.  I'll try to find out -- tbm

--- a/drivers/net/tulip/tulip_core.c	2007-05-21 11:12:45.000000000 +0000
+++ b/drivers/net/tulip/tulip_core.c	2007-05-21 11:13:10.000000000 +0000
@@ -1490,8 +1490,8 @@
                     (PCI_SLOT(pdev->devfn) == 12))) {
                        /* Cobalt MAC address in first EEPROM locations. */
                        sa_offset = 0;
-		       /* Ensure our media table fixup get's applied */
-		       memcpy(ee_data + 16, ee_data, 8);
+		       /* No media table either */
+		       tp->flags &= ~HAS_MEDIA_TABLE;
                }
 #endif
 #ifdef CONFIG_GSC
